master/reference/kube-controllers/configuration.md

```diff
@@ -45,11 +45,12 @@ configure API access if needed.
 
 The following environment variables can be used to configure the {{site.prodname}} Kubernetes controllers.
 
-| Environment   | Description | Schema |
-| ------------- | ----------- | ------ |
-| `ENABLED_CONTROLLERS` | Which controllers to run | namespace, node, policy, serviceaccount, workloadendpoint |
-| `LOG_LEVEL`     | Minimum log level to be displayed. | debug, info, warning, error |
-| `KUBECONFIG`    | Path to a kubeconfig file for Kubernetes API access | path |
+| Environment   | Description | Schema | Default |
+| ------------- | ----------- | ------ | -------
+| `ENABLED_CONTROLLERS` | Which controllers to run | namespace, node, policy, serviceaccount, workloadendpoint | policy,namespace,serviceaccount,workloadendpoint,node
```

Comment:
The listed default here: ```policy,namespace,serviceaccount,workloadendpoint,node```
does not match the default described on line 55...  ```policy, namespace, workloadendpoint, serviceaccount```

Why does line 55 exist when that info is in the table here?

Reply:
Interesting...the default displayed in the table is actual value (in code) used in the manifest while the section below is listing them in human English.

Reply:
Hmm.  I'd still expect the human english to match the table (and code) :)